destalinator.py

```diff
@@ -42,7 +42,7 @@ def add_slack_channel_markup_item(self, item):
         return self.slacker.add_channel_markup(item.group(1))
 
     def add_slack_channel_markup(self, text):
-        marked_up = re.sub(r"\#(\w+)", self.add_slack_channel_markup_item, text)
+        marked_up = re.sub(r"\#([\w-]+)", self.add_slack_channel_markup_item, text)
```

Comment:
Though this reveals some underlying weaknesses of using this character class:
* Uppercase letters are _not_ supported in channel names, but _are_ included in `\w`
* `\w` isn't the best explanation here; we should likely use a full character class to describe the allowable set, e.g. `[a-z_-]`, once I hear back from the Slack team on what “non-Latin characters” might mean. 😄